bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_tau_NLO/HeavyNeutrino_trilepton_mNltmW_V-0p01_tau_NLO/HeavyNeutrino_trilepton_V-0p01_tau_NLO_madspin_card.dat

```diff
@@ -0,0 +1,7 @@
+set ms_dir ./madspingrid
+set Nevents_for_max_weigth 250
+set max_weight_ps_point 400
+set spinmode none
+set max_running_process 1
+decay n1 > v ldecay ldecay
+launch
```

Comment:
Should we remove this mN<mW cards for NLO? As you are going to use LO for these mass points?

Reply:
I will remove them, we are now planning to use LO.